Web Development/Resources/README.md

```diff
@@ -3,7 +3,11 @@
 - [The Complete 2020 Web Development Bootcamp](https://www.udemy.com/course/the-complete-web-development-bootcamp/)
 - [The Web Developer Bootcamp](https://www.udemy.com/course/the-web-developer-bootcamp/)
 - [10 Web Development YouTube Channels You Probably Didn't Know About](https://dev.to/ryandsouza13/10-web-development-youtube-channels-you-probably-didn-t-know-about-4o37?utm_source=digest_mailer&utm_medium=email&utm_campaign=digest_email)
+- [Web developement courses on coursera](https://www.coursera.org/courses%3Fquery%3Dweb%2520development&ved=2ahUKEwiLzJXwlKLsAhV8wTgGHfi6AG8QFjAJegQIIBAH&usg=AOvVaw3RaH7LlNPk66swGThS5zNv)
```

Comment:
This link does not work. Even if it did, why would someone use this link to search Web Development Courses on Coursera when they can do that already on Coursera itself.